### ISSUE TITLE
Message to DKHamsing

### ISSUE THREAD

**Issue by philipturner** [dkhamsing/open-source-ios-apps#989]
dkhamsing, I don't have any other way to contact you besides a GitHub issue.

I made a PR for my framework, [ARHeadsetKit](https://github.com/philipturner/ARHeadsetKit), to go onto matteocrippa's [awesome-swift](https://github.com/matteocrippa/awesome-swift) list. There are multiple people who have approved PR's in the past, but you are the most active. At the moment, ARHeadsetKit has 14 out of 15 stars needed for submission.

Would you mind taking a look at my PR on awesome-swift? If it is still at 14 stars when you look at it, could you star it just to get it to qualify? I know you have zero stars on your profile, so you could just un-star it when it reaches 16.

I wish I could add ARHeadsetKit to your open-source-ios-apps list alongside AR MultiPendulum, but unfortunately it is a framework, not an app. Hopefully other apps made with it will become part of your list!

**Comment by philipturner** [comment]
Yes! Just a minute ago, and right before you looked at it!

**Comment by philipturner** [comment]
I also added a PR to add ARHeadsetKit to [vsouza/awesome-ios](https://github.com/vsouza/awesome-ios/commits/master). You are a maintainer there, and have approved multiple pull requests. I'm hoping you can let me skip the 1-month gap between @vsouza's strings of commits (the last one was early August).

**Comment by philipturner-testing** [comment]
_Please don't block this account from your repository. I am fully conscious that it looks like I am spamming again._

I apologize for opening irrelevant pull requests while trying to get in contact with you. It wasn't the best way to get in touch. In fact, you don't want to be messaged at all by anyone (I just did some research to find that out). I had no idea that was the case when the infamous triple "Message to DKHamsing" happened.

I got the wrong impression about that fact from my first "Message to DKHamsing", where you responded to my question instead of asking me to stop. I got thrown off with your vastly different response to the second message. The final one was a poorly executed attempt to resolve the situation. I ended up blocked from commenting on issues for this repo. Sorry for misunderstanding you, and I won't do that again. Please trust me on this.

A month ago, I opened an issue about the still-unsolved problem with duplication on the open-source-ios-apps website. After that, I let you know when it popped up again. I can't help you any more with the duplication bug, now that my main account is blocked.

Could you please unblock me from the repository so I can let you know if the website has the bug again? If not, please at least refrain from blocking my alt account. Any other person looking at this conversation would associate the alt with me spamming, and (again) I'm really not trying to do that.

I sent this message assuming that you are okay with comments on closed issues. If that is not the case, please don't block my alt account for that misunderstanding :-)

---

The whole reason I tried contacting you is that you are one of the very few (if not the only) maintainers of several awesome lists who responds back within a day. If you forgive me for the trouble I caused, would this thread be fine for communication between us? Since it's closed, it won't distract other people trying to view new issues.

I won't bother you for help with pull requests for awesome lists anymore. You'll probably just reject all of them since I caused you so much trouble. I'm just asking if it's okay to keep open this line of communication for anything else that might happen in the future. Although if you are okay with me using this thread for help on PRs, please let me know.

**Comment by philipturner-testing** [comment]
Also (in the case you unblock me) could I (or you) rename two of the issues that all say "Message to DKHamsing", so that people don't skim over the issue history and think I was spamming you? I already had that happen with @herrbischoff.
